src/Compiler/Checking/CheckDeclarations.fs

```diff
@@ -4047,7 +4051,13 @@ module TcDeclarations =
             let members = 
                 let membersIncludingAutoProps = 
                     cspec |> List.filter (fun memb -> 
-                      match memb with 
+                      match memb with
+                      | SynMemberDefn.ImplicitCtor(ctorArgs= SynSimplePats.SimplePats(pats, _)) when  not pats.IsEmpty && reportErrorOnStaticClass ->
+                            for pat in pats do
+                                 errorR(Error(FSComp.SR.chkErrorOnStaticClasses "constructor arguments are", pat.Range))
+                            false
+                      | SynMemberDefn.Member(SynBinding(valData = SynValData(Some memberFlags, _, _)), m)  when memberFlags.MemberKind = SynMemberKind.Constructor && reportErrorOnStaticClass ->
+                          errorR(Error(FSComp.SR.chkErrorOnStaticClasses "additional constructors are", m)); false
```

Comment:
Bear in mind that all the text in FsComp.txt gets localized, whereas strings in compiled code are not.
I think it is OK to use parametrized templates, but then the embedded snippets should be as easy as possible, and typically not affect the form of the sentence even if different languages.

Reply:
I have updated to use 2 different error numbers so this is not longer the case :)